apiserver/allfacades.go

```diff
@@ -135,7 +134,7 @@ func AllFacades() *facade.Registry {
 	reg("Controller", 3, controller.NewControllerAPI)
 	reg("Deployer", 1, deployer.NewDeployerAPI)
 	reg("DiskManager", 2, diskmanager.NewDiskManagerAPI)
-	reg("Firewaller", 3, firewaller.NewFirewallerAPI)
+	reg("Firewaller", 3, firewaller.NewStateFirewallerAPI)
```

Comment:
bump the version? can we claim support for this new method? maybe it needs to be added to the facade only when the feature flag is set?